studio/schemas/faq.ts

```diff
@@ -0,0 +1,20 @@
+import { HelpCircleIcon } from "@sanity/icons";
+
+export default {
+  title: "FAQ",
+  name: "faq",
+  type: "document",
+  icon: HelpCircleIcon,
+  fields: [
+    {
+      title: "Pregunta",
+      name: "question",
+      type: "string",
+    },
+    {
+      title: "Respuesta",
+      name: "answer",
+      type: "string",
```

Comment:
@TonyMckes para la respuesta usemos el tipo block ya que puede que necesiten poner links, o cosas por el estilo.
https://www.sanity.io/docs/block-type

Reply:
No se si ademas consideras necesario agregarle el `required()` @nykka.
Just in case

Reply:
> No se si ademas consideras necesario agregarle el `required()` @nykka. Just in case

Utilizar `required()` rompe el build de Next.js, se me ocurren dos soluciones, pero igual estamos si los schemas quedaran sin cambios

Reply:
Quedamos en que por ahora no agregamos ninguna validación. Tenemos un issue separado para investigarlo #50